modules/renter.go

```diff
@@ -52,6 +52,7 @@ type FileInfo struct {
 	Filesize       uint64            `json:"filesize"`
 	Available      bool              `json:"available"`
 	Renewing       bool              `json:"renewing"`
+	Redundancy     float64           `json:"redundancy"`
 	UploadProgress float64           `json:"uploadprogress"`
```

Comment:
They are redundant. But user's expect to see an upload progress. We could calculate the upload progress from the redundancy in the client, but then the target redundancy would need to be known to the client.